CHANGELOG.md

```diff
@@ -4,12 +4,58 @@
 
 ### API changes
 
+#### Added
+
+- added `dc_context_t*   dc_context_new_closed(const char* dbfile)`, `int  dc_context_open(dc_context_t *context, const char* passphrase)`, `int  dc_context_is_open(dc_context_t *context)` and `uint32_t  dc_accounts_add_closed_account(dc_accounts_t* accounts)` #2956
+
 - added `DC_MSG_WEBXDC`, `dc_send_webxdc_status_update()`,
   `dc_get_webxdc_status_updates()`, `dc_msg_get_webxdc_blob()`,
   `dc_msg_get_webxdc_info()`, and `DC_EVENT_WEBXDC_STATUS_UPDATE` #2826
+
+- Add `dc_msg_force_plaintext()` API for bots #2847
+
+- allow to remove quotes on drafts `dc_msg_set_quote(msg, NULL)` #2950
+
+#### Removed
+
 - Removed `mvbox_watch` option. #2906
   It is automatically enabled whenever `mvbox_move` is enabled.
+- Removed `inbox_watch` option #2922
 
+#### Deprecated
+- deprecated `os_name` in `dc_context_new(const char* os_name, const char* dbfile, const char* blobdir)`, now you should pass `NULL` or an empty string.
+
+### Added
+- Add imap table to keep track of message UIDs #2909
+- add hop_info (#2751)
+- Start making it possible to write to mailing lists (#2736)
```

Comment:
```suggestion
- Start making it possible to write to mailing lists #2736
```

Reply:
... there are also brackets in some other lines ... we should just be consistent.